### PR TITLE
Create layer 'apache'

### PR DESCRIPTION
**Pull request by gderber**
This is a simple layer, based HEAVILY on the nginx layer.  In fact, almost an exact duplicate replacing nginx with apache.

I wasn't 100% certain what image to use for apache.png.  I used the one on the front page of their website.

### CODE REVIEW

File: layers/+tools/apache/README.org [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+* Table of Contents                     :TOC_5_gh:noexport:
+- [[#description][Description]]
+  - [[#features][Features:]]
+- [[#install][Install]]
+
+* Description
+This layer adds support for configuring the [[https://httpd.apache.org/][Apache]] web server.
+
+** Features:
+- Syntax highlighting of nginx configuration files via [[https://github.com/emacs-php/apache-mode][apache-mode]].
```
Comment (smile13241324) [inline layers/+tools/apache/README.org]: This should be apache configuration files I guess

File: layers/+tools/apache/packages.el [hunk 2]
```diff
@@ -0,0 +1,27 @@
+;;; packages.el --- nginx layer packages file for Spacemacs.
```
Comment (smile13241324) [inline layers/+tools/apache/packages.el]: nginx -> apache

**Review by smile13241324 — CHANGES_REQUESTED** [review]
I have made a quick review and found some smaller issues you need to fix before this can be merged.

File: layers/+tools/apache/packages.el (hunk 2 above)
Comment (gderber) [reply layers/+tools/apache/packages.el]: I've made these fixes in a new commit.  I wasn't sure if you'd want a squashed commit.  If so let me know.

**Review by smile13241324 — APPROVED** [review]

**Pull request merged**